app/models/setting/katello.rb

```diff
@@ -12,6 +12,9 @@ def self.load_defaults
         self.set('katello_default_ptable', N_("Default partitioning table for new Operating Systems"), 'Kickstart default'),
         self.set('content_action_accept_timeout', N_("Time in seconds to wait for a Host to pickup a remote action"), 20),
         self.set('content_action_finish_timeout', N_("Time in seconds to wait for a Host to finish a remote action"), 3600),
+        self.set('restrict_composite_view', N_("If set to true, a composite content view may not be published or "\
```

Comment:
@ehelms, Correct, this enhancement is initially being done as app wide based upon user request.  We can further enhance it in the future, if we find it to be useful to have lower levels of control.